assets/js/components/data/index.js

```diff
@@ -255,10 +226,48 @@ const dataAPI = {
 			// Resolve any returned data requests, then re-request the remainder after a pause.
 		} ).catch( ( err ) => {
 			// Handle the error and give up trying.
-			console.log( 'error', err ); // eslint-disable-line no-console
+			console.warn( 'Error caught during combinedGet', err ); // eslint-disable-line no-console
 		} );
 	},
 
+	handleWpError( error ) {
+		const { code, data } = error;
+
+		if ( ! code || ! data ) {
+			return;
+		}
+
+		// eslint-disable-next-line no-console
+		console.warn( 'WP Error in data response', error );
```

Comment:
Maybe we could do an early return here if `! data.reason`? No point executing further then.